src/vs/workbench/contrib/chat/browser/chatEditing/chatEditingSession.ts

```diff
@@ -793,7 +794,12 @@ export class ChatEditingSession extends Disposable implements IChatEditingSessio
 				didComplete = true;
 				sequencer.queue(async () => {
 					if (!this.isDisposed) {
-						await this._acceptTextEdits(resource, [], true, responseModel);
+						const entry = await this._getOrCreateModifiedFileEntry(resource, this._getTelemetryInfoForModel(responseModel)) as IModifiedFileEntry;
+						if (entry.acceptAgentNotebookEdits) {
+							await this._acceptNotebookEdits(resource, [], true, responseModel);
```

Comment:
Not sure this is the right way/, but seems to make sense, as with notebooks the last operation must be the edit with `isDone = true`